src/CommandingOfficers/Meridian.java

```diff
@@ -75,150 +73,139 @@ public Meridian(GameScenario.GameRules rules)
     arty.possibleActions.add(new TransformLifecycle.TransformFactory(tank, "~TANK"));
 
     addCommanderAbility(new ChangeAndFlow(this));
-    addCommanderAbility(new VehicularCharge(this));
+    addCommanderAbility(myVehicularCharge);
```

Comment:
Why does Vehicular Charge need to be created in advance?

Reply:
It doesn't, but it's fewer lines of code and equivalent meaning this way.
Meridian keeps a reference to it around so she can mark all of her own units she's currently debuffing

I considered pushing her markings into some sort of generic StateTracker that just handles arbitrary markings, but that isn't something the current implementation of unit markings/state trackers lends itself to (deliberately; whether that's good in practice is a valid consideration)